src/group_theory/perm/fin.lean

```diff
@@ -95,6 +96,27 @@ begin
   { rw fin_rotate_succ, simp [ih, pow_succ] },
 end
 
+@[simp] lemma support_fin_rotate {n : ℕ} : support (fin_rotate (n + 2)) = finset.univ :=
```

Comment:
Does the `n + 2` make it hard to `rw` with this lemma? Would it be easier to use this lemma if there was instead an `2 ≤ n` (or 1 < n) hypothesis?

Reply:
For this one, I'm on the fence, I'll have to check.

Reply:
In my proof of simplicity of A5, I have no difficulty rewriting, and in fact I think it's easier this way. I don't think it'll come up in the proof of simplicity for larger A6. I guess it just depends on whether we think things like this will come up more frequently for explicit numerals or for variables.

Reply:
I think it's fine to have this as a simp-lemma, and add `support_fin_rotate_of_le` below it.